selfdrive/car/gm/interface.py

```diff
@@ -45,7 +49,10 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, disa
     ret = CarInterfaceBase.get_std_params(candidate, fingerprint)
     ret.carName = "gm"
     ret.safetyConfigs = [get_safety_config(car.CarParams.SafetyModel.gm)]
-    ret.pcmCruise = False  # stock cruise control is kept off
+    ret.pcmCruise = False  # For ASCM, stock cruise control is kept off (but not ACC)
```

Comment:
The existing comment seemed to need expansion, but I could use help.
* What does PCM stand for / mean?
* What exactly is kept off - standard CC? I have gathered that ACC is kept enabled and the car thinks it is managing the ACC
* Do cars with an ASCM + ACC have separate CC and ACC?

I discovered that flipping this from False to True took care of all the little bits I had been manually commenting out in controlsd... Happy that it works but still a little confused